src/components/Calendar/Event/CreateEvent.js

```diff
@@ -1,21 +1,21 @@
 import * as React from 'react';
 import {Radio, RadioGroup, Typography} from "@mui/material";
 import TextField from '@mui/material/TextField';
-import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker';
+import {MobileDatePicker} from '@mui/x-date-pickers/MobileDatePicker';
 import {PrimaryButton2} from '../../CustomMUIComponents/CustomButtons';
- import {LocalizationProvider} from "@mui/x-date-pickers";
+import {LocalizationProvider} from "@mui/x-date-pickers";
 import {AdapterDayjs} from "@mui/x-date-pickers/AdapterDayjs";
 import Switch from "@mui/material/Switch";
 import FormControlLabel from "@mui/material/FormControlLabel";
 import FormControl from "@mui/material/FormControl";
 import axios from "axios";
 import GetAuthentication from "../../Authentication/Authentification";
-import {  EventForm } from '../Custom/CommonInputEventForm';
-import { Stack } from '@mui/system';
-
+import {EventForm} from '../Custom/CommonInputEventForm';
+import {Stack} from "@mui/material";
```

Comment:
Duplicate import. Import from `"@mui/material"` exists on line 2. Just add `Stack` to the `{...}` on line 2.